README.md

```diff
@@ -33,6 +33,29 @@ CLI command syntax for invoking ```duploctl```
 duploctl <resource> <command> <args...>
 ```
 
+### Example Usages
+Configure `duploctl` access with environment variables:
+```sh
+export DUPLO_HOST=https://example.duplocloud.net
+export DUPLO_TOKEN=AQAAA...
+export DUPLO_TENANT=dev01
+```
+
+List the services in a tenant:
+```sh
+duploctl service list
+```
+
+Get AWS Console URL:
+```sh
+duploctl jit aws
+```
+
+Get Kubernetes config:
```

Comment:
Nobody should run this directly. This is more for the exec inside of a kubeconfig. 
Change this to
```
duploctl jit update_kubeconfig
```

Reply:
sure

Reply:
@jmaggio  Can you please add this to wiki as well.

Reply:
The scope here is just to get a quick reference that people can immediately see when coming to this repo in github, wiki is more detailed and out of scope for this effort @akshay-duplo